tests/backup/backup_helper.go

```diff
@@ -3996,6 +3998,20 @@ func IsClusterPresent(clusterName string, ctx context.Context, orgID string) (bo
 	return false, nil
 }
 
+// GetConfigObj reads the configuration file and returns a BackupCloudConfig object.
+func GetConfigObj() (*backup.BackupCloudConfig, error) {
+	var config *backup.BackupCloudConfig
+	cm, err := core.Instance().GetConfigMap(cloudCredConfigMap, "default")
+	if err != nil {
+		log.Warnf("Error reading config map, continuing the run ,in case if you are running on cloud platform make sure to add configmap : %v", cloudCredConfigMap)
+		return config, nil
+	}
```

Comment:
This is not what we discussed. We have to have a check at the beginning itself.
We are going to list all the config maps in default namespace and check if `cloudCredConfigMap` is even created. If it is created, we will go with the usual flow.
Else, we will just log a warning and return nil struct and move on. This will keep the on prem runs unaffected.